x-pack/plugins/osquery/public/live_queries/form/index.tsx

```diff
@@ -50,20 +65,24 @@ const LiveQueryFormComponent: React.FC<LiveQueryFormProps> = ({
     }
   );
 
+  const formSchema = {
+    query: {
+      type: FIELD_TYPES.TEXT,
+      validations: [{ validator: queryValidator }],
+    },
+  };
+
   const { form } = useForm({
     id: FORM_ID,
-    // schema: formSchema,
+    schema: formSchema,
     onSubmit: (payload) => {
       return mutateAsync(payload);
     },
     options: {
       stripEmptyFields: false,
     },
     defaultValue: defaultValue ?? {
-      query: {
-        id: null,
```

Comment:
we want to keep the `id`, so in the future if the user wants to use saved query then we can correlate the results

Reply:
we don't currently have the support for saved queries, so i ripped that out and we can add it back in when we add support for it.

Reply:
talked about this on slack, going to pass on reintroducing the old id code.